lib/WebSocket.js

```diff
@@ -2,6 +2,7 @@ const  WebSocket = require('ws');
 const url = require('url');
 const DockerWrapper = require('../lib/DockerWrapper');
 const App = require('../server/models').App;
+const fetch = require('node-fetch');
```

Comment:
Ah so this is essentially a Node implementation of the browser's `fetch` API? Nice.

Reply:
Yep, it was just what the doctor ordered